audbackend/core/artifactory.py

```diff
@@ -123,3 +127,18 @@ def _versions(
         r"""Versions of a file."""
         group_id = audfactory.path_to_group_id(folder)
         return audfactory.versions(self.host, self.repository, group_id, name)
+
+    _non_existing_path_error = (RuntimeError, requests.exceptions.HTTPError)
+    r"""Error expected for non-existing paths.
+
+    If a user has no permission to a given path
+    or the path does not exists :func:`audfactory.path`
+    might return a
+    ``RuntimeError: 404 page not found``
+    or
+    ``requests.exceptions.HTTPError: 403 Client Error``
+    error,
+    which might depend on the instaleld ``dohq-artifactory``
```

Comment:
```suggestion
    which might depend on the installed ``dohq-artifactory``
```

Reply:
Sorry, too late.

As we need this fix as fast as possible I decided to go without review.